Case sensitive file systems brake on require

Throughout the codebase several instances of require("") use lowercase filenames, even though the real filename has a capital letter in it. On macOS this is totally fine, since the fs is case insensitive. But on ubuntu which is case sensitive, we get the following error:

```
Error: Cannot find module './base'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/ubuntu/XXXXXX/node_modules/paymentwall/lib/Widget.js:3:12)
```

This at line 3 in Widget.js
`var Base = require('./base')`, should be `var Base = require('./Base')`

--- a/lib/ApiObject.js
+++ b/lib/ApiObject.js
@@ -1,6 +1,6 @@
 'use strict';
 
-var Base = require('./base'),
+var Base = require('./Base'),
 	util = require('util'),
 	querystring = require('querystring');
 
@@ -13,7 +13,7 @@ util._extend(ApiObject.prototype, {
 
 	checkProjectEnv: function(){
 		var reg = /_+/;
-		return reg.test(this.secretKey); 
+		return reg.test(this.secretKey);
 	},
 
 	createPostOptions: function(url, path, method){
@@ -28,7 +28,7 @@ util._extend(ApiObject.prototype, {
 		  	}
 		};
 		return post_options;
-	}, 
+	},
 
 	createRequest: function(type, additional_path){
 		var url;
@@ -43,9 +43,9 @@ util._extend(ApiObject.prototype, {
 			url = this.BRICK_BASE_URL;
 			method = "Post";
 			if (type === "onetimetoken") {
-				path = this.BRICK_ONETIMETOKEN_PATH; 
+				path = this.BRICK_ONETIMETOKEN_PATH;
 			} else if (type === "charge"){
-				path = this.BRICK_CHARGE_PATH+additional_path; 
+				path = this.BRICK_CHARGE_PATH+additional_path;
 			} else if (type === "subscription"){
 				path = this.BRICK_SUBSCRIPTION_CHARGE_PATH+additional_path;
 			};

--- a/lib/Pingback.js
+++ b/lib/Pingback.js
@@ -1,7 +1,7 @@
 'use strict';
 
-var Base = require('./base'),
-	Product = require('./product'),
+var Base = require('./Base'),
+	Product = require('./Product'),
 	util = require('util'),
 	Signature = require('./Signature/Pingback'),
 	querystring = require('querystring');
@@ -28,7 +28,7 @@ function Pingback(parameters, ipAddress, pingbackForBrick) {
 util.inherits(Pingback, Base);
 
 util._extend(Pingback.prototype, {
-	
+
 	validate: function(skipIpWhitelistCheck) {
 		var pingbackForBrick = this.pingbackForBrick;
 		var skipIpWhitelistCheck = skipIpWhitelistCheck || false;
@@ -65,10 +65,10 @@ util._extend(Pingback.prototype, {
 		} else { // API_CART
 			signatureParams = ['uid', 'goodsid', 'type', 'ref'];
 			this.parameters['sign_version'] = this.SIGNATURE_VERSION_2;
-		} 
+		}
 
 		if (!this.parameters['sign_version'] || this.parameters['sign_version'] === this.SIGNATURE_VERSION_1) {
-			
+
 			var ref = this;
 
 			signatureParams.forEach(function(field) {
@@ -79,7 +79,7 @@ util._extend(Pingback.prototype, {
 		} else {
 			signatureParamsToSign = this.parameters;
 		}
-		
+
 		var signatureCalculated = Signature.calculateSignature(signatureParamsToSign, this.getSecretKey(), this.parameters['sign_version']);
 		var signaturePassed = (this.parameters['sig'] !== undefined) ? this.parameters['sig'] : null;
 		return signaturePassed === signatureCalculated;
@@ -114,7 +114,7 @@ util._extend(Pingback.prototype, {
 		}
 
 		var ref = this;
-		
+
 		if (typeof ref.parameters !== 'object') {
 			ref.parameters = querystring.parse(ref.parameters);}
 

--- a/lib/Widget.js
+++ b/lib/Widget.js
@@ -1,7 +1,7 @@
 'use strict';
 
-var Base = require('./base'),
-	Product = require('./product'),
+var Base = require('./Base'),
+	Product = require('./Product'),
 	util = require('util'),
 	querystring = require('querystring'),
 	Signature = require('./Signature/Widget');
@@ -32,7 +32,7 @@ util._extend(Widget.prototype, {
 
 		if (this.getApiType() === this.API_GOODS) {
 			if (this.products) {
-				if (productsNumber === 1) {	
+				if (productsNumber === 1) {
 
 					var product = this.products[0];
 					var postTrialProduct = null;
@@ -46,7 +46,7 @@ util._extend(Widget.prototype, {
 					params['ag_name'] = product.getName();
 					params['ag_external_id'] = product.getId();
 					params['ag_type'] = product.getType();
-					
+
 					if (product.getType() === Product.TYPE_SUBSCRIPTION) {
 
 						params['ag_period_length'] = product.getPeriodLength();

